kubernetes/ama-logs.yaml

```diff
@@ -709,16 +709,16 @@ spec:
               name: host-log
             - mountPath: /etc/kubernetes/host
               name: azure-json-path
-            - mountPath: /etc/omsagent-secret
-              name: omsagent-secret
+            - mountPath: /etc/ama-logs-secret
+              name: ama-logs-secret
               readOnly: true
             - mountPath: /etc/config
-              name: omsagent-rs-config
+              name: ama-logs-rs-config
             - mountPath: /etc/config/settings
               name: settings-vol-config
               readOnly: true
             - mountPath: /etc/config/settings/adx
-              name: omsagent-adx-secret
+              name: ama-logs-adx-secret
```

Comment:
we will have to validate the ADX scenario to ensure everything works as expected.